MotionMark/tests/bouncing-particles/resources/bouncing-canvas-particles.js

```diff
@@ -22,26 +22,43 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
-BouncingCanvasParticle = Utilities.createSubclass(BouncingParticle,
-    function(stage, shape)
+
+class BouncingCanvasParticle extends BouncingParticle {
+    static clips;
+    constructor(stage, shape)
     {
-        BouncingParticle.call(this, stage);
+        BouncingCanvasParticle.clips = {
```

Comment:
This can be moved with the definition above. There is no need to define a static member and then construct it in the constructor.